### PR TITLE
added functionalities

### PR DESCRIPTION
**Pull request by Dhruv0201**
Added Some functionalities merge it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (MilanPadhariya) [inline .gitignore:1]: why this node modules folder in gitignore?

File: controller/deskController.js [hunk 2]
```diff
@@ -23,6 +23,28 @@ exports.getAllDesks=async(req,res)=>
...
+})
+next();
+}
+catch(err){
+    res.status(400).json({
+        status:"Fail"
+    })
+}
+
+}
```
Comment (MilanPadhariya) [inline controller/deskController.js:43]: indentation review

File: package.json [hunk 3]
```diff
@@ -5,17 +5,20 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```
Comment (MilanPadhariya) [inline package.json:8]: during development phase it is nodemon is better but during hosting node should be used

**Review by MilanPadhariya — CHANGES_REQUESTED** [review]
reviewed

**Review by MilanPadhariya — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Dhruv0201) [reply .gitignore:1]: was trying some experiment with gitignore then i forgot to change back.. i apologize for silly mistake

File: package.json (hunk 3 above)
Comment (Dhruv0201) [reply package.json:8]: okay !!!

**Pull request merged**